engines/ags/detection_tables.h

```diff
@@ -346,13 +346,13 @@ const PlainGameDescriptor GAME_NAMES[] = {
 	{ "onironauta", "Onironauta" },
 
 	// AGDI games. They get their own grouping because they're just that awesome
-	{ "kq1agdi", "King's Quest I: Quest for the Crown Remake" },
+	{ "kq1agdi", "King's Quest: Quest for the Crown Remake" },
```

Comment:
AGDI uses the "I" on their website, so I think it should be kept
https://www.agdinteractive.com/games/kq1/index.html

Reply:
For conventions and just a overall guide for naming I have based it all off of the box title and for this engine the box art has this title: "https://www.mobygames.com/game/dos/kings-quest-quest-for-the-crown/cover-art/gameCoverId,27584/" I cant just use the website because the different engines came with different names as they were re-released for different games

Reply:
Space quest is a good example somtimes its space quest: chapter I: The Sarien Encounter adn on other engines its space quest I: Roger Wilco in the Sarien Encounter https://www.mobygames.com/game/space-quest-i-roger-wilco-in-the-sarien-encounter  and https://www.mobygames.com/game/amiga/space-quest-chapter-i-the-sarien-encounter

Reply:
The AGDI games are fan-made remakes, so they should use the name they were released with, not any official name of the original game

Reply:
> The AGDI games are fan-made remakes, so they should use the name they were released with, not any official name of the original game

Exactly. Also, that boxart is most likely user-made (or maybe provided by tierra for convenience as a picture/pdf) since that remake was never released on disc.
The kq1agdi ID targets both Tierra and AGDI releases, but since most players are going to use the latter, I'd stick to AGDI naming.

Reply:
Okay I'll go change those thank you guys